platform_orchestrator/src/main/java/com/override/service/VerificationService.java

```diff
@@ -37,16 +38,24 @@ public class VerificationService {
     @Autowired
     private PersonalDataMapper personalDataMapper;
 
-    @CachePut(value="codeCallSecurity")
+    @Autowired
+    private CacheManagerSettingsGenerationService cacheManagerSettingsGenerationService;
+
+    @PostConstruct
+    private void setCacheSettings() {
+        cacheManager.setCacheSpecification(cacheManagerSettingsGenerationService.getVerificationCacheManagerSpecification());
+    }
```

Comment:
https://stackoverflow.com/questions/49885064/is-it-possible-to-set-a-different-specification-per-cache-using-caffeine-in-spri